### PR TITLE
Fixed passing array values in query

### LINKED ISSUE #282: Invalid params cause app crash
If we use an invalid filter in the URL that was supposed to be an array, e.g. `/?searchValue=&technologies=Java`, the app will crash and present a blank page

### PR DESCRIPTION
**Pull request by Naapperas**
Closes #282

### CODE REVIEW

File: src/components/HomePage/SearchArea/useUrlSearchParams.js [hunk 1]
```diff
@@ -75,24 +79,33 @@ export default ({
...
+        /* istanbul ignore else */
         if (setShowJobDurationSlider)
             setShowJobDurationSlider(showJobDurationSlider);
 
     }, [changeURLFilters, location, queryParams, setShowJobDurationSlider]);
 
     const actualSetFields = useCallback((fields) => {
 
+        // eslint-disable-next-line no-param-reassign
+        fields = ensureArray(fields);
```
Comment (BrunoRosendo) [inline src/components/HomePage/SearchArea/useUrlSearchParams.js:91]: Why not just create a new variable? The eslint error probably exists for a valid reason

File: src/components/HomePage/SearchArea/useUrlSearchParams.js [hunk 2]
```diff
@@ -75,24 +79,33 @@ export default ({
...
+        /* istanbul ignore else */
         if (setFields)
             setFields(fields);
 
     }, [changeURLFilters, location, queryParams, setFields]);
 
     const actualSetTechs = useCallback((technologies) => {
 
+        // eslint-disable-next-line no-param-reassign
+        technologies = ensureArray(technologies);
```
Comment (BrunoRosendo) [inline src/components/HomePage/SearchArea/useUrlSearchParams.js:104]: Same as above

File: src/components/HomePage/URLSearchParamsParser.js [hunk 3]
```diff
@@ -7,6 +7,10 @@ import { useSelector } from "react-redux";
 import useOffersSearcher from "./SearchResultsArea/SearchResultsWidget/useOffersSearcher";
 import { SearchResultsConstants } from "./SearchResultsArea/SearchResultsWidget/SearchResultsUtils";
 
+import { ensureArray } from "../../utils";
+
+// TODO: should write tests for this
```
Comment (BrunoRosendo) [inline src/components/HomePage/URLSearchParamsParser.js:12]: I think they are done?

**Review by BrunoRosendo — CHANGES_REQUESTED** [review]
I left some comments regarding disabling eslint and code coverage

File: src/components/HomePage/URLSearchParamsParser.js (hunk 3 above)
Comment (Naapperas) [reply src/components/HomePage/URLSearchParamsParser.js:12]: They are, but in the `HomePage.spec.js` flle

**Review by BrunoRosendo — APPROVED** [review]
good job

**Pull request merged**